libpod/container.go

```diff
@@ -534,6 +538,89 @@ func (c *Container) refresh() error {
 	return nil
 }
 
+// GetCommand returns command to be executes inside of container
+// ToDO need to add Command handling to libpod
+func (c *Container) GetCommand() string {
```

Comment:
Should also be able to get this from the OCI spec, which we do store.

Reply:
I think I'd prefer if we get rid of this and GetAnnotations() and just have people calling libpod get the spec and get the things they need out of it

Reply:
Fixed